src/core/Global.ttslua

```diff
@@ -645,6 +646,20 @@ function emptyChaosBag()
   end
 end
 
+-- move all regular chaos tokens that are on the table to the chaos bag (e.g. sealed on cards)
+function findAllChaosTokens()
+  local chaosbag = findChaosBag()
+  for _, obj in ipairs(getObjects()) do
```

Comment:
If a user has other tokens on the table, this will put them in the bag.  I don't know about other groups, but when I have a campaign with a highly mutable bag through scenarios I tend to leave the extra copies on the table.

Reply:
Good point. Now it will only return sealed tokens